core/cmd.go

```diff
@@ -57,6 +57,22 @@ func (ch *CommandHandler) GetAppVersion() string {
 	return AppVersion.Original()
 }
 
+func (ch *CommandHandler) CheckForUpdate() bool {
+	latestVersion, err := ch.browser.GetLatestAppVersion()
+	if err != nil {
+		log.Println(err)
+		return false
+	}
+
+	if AppVersion.LessThan(latestVersion) {
```

Comment:
Could return early and skip the else? might make things cleaner and would avoid the ugly double call